bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/SystemFollowProfile.java

```diff
@@ -53,12 +50,7 @@ public ProfileTypeUID getProfileTypeUID() {
 
     @Override
     public void onStateUpdateFromItem(State state) {
-        if (!(state instanceof Command)) {
-            logger.debug("The given state {} could not be transformed to a command", state);
-            return;
-        }
-        Command command = (Command) state;
-        callback.handleCommand(command);
+        callback.handleUpdate(state);
```

Comment:
I agree with @pfink, the follow profile will forward state updates from an item to the following item as commands. Please revert.